Code/Tools/AssetProcessor/native/unittests/FileWatcherUnitTests.cpp

```diff
@@ -300,10 +300,10 @@ namespace FileWatcherTests
         {
             QString filename = QDir(m_assetRootPath).absoluteFilePath(QString("test%1.tif").arg(fileIndex));
             filename = QDir::toNativeSeparators(filename);
-            EXPECT_STREQ(m_filesAdded[fileIndex].toUtf8().constData(), filename.toUtf8().constData());
+            EXPECT_STREQ(m_filesAdded[static_cast<unsigned int>(fileIndex)].toUtf8().constData(), filename.toUtf8().constData());
```

Comment:
nit: Don't feel the need to change this now but I think it's generally preferred to use `uint32_t` as opposed to `unsigned int` (much less typing for one thing 😅)

Reply:
The warning says is about a cast from` unsigned long` to **signed** `int`, not `unsigned int`.

Reply:
I tried `unsigned int` first to cover more numbers from `unsigned long` and since it didn't complain anymore I left it with unsigned. Maybe the compiler is then converting it to signed, but it's not complaining anymore.